### ISSUE TITLE
@5.1: Cannot find module 'babel-runtime/core-js/is-iterator'

### ISSUE THREAD

**Issue by deepsweet** [babel/babel#1243]
required by `babel-runtime/helpers/sliced-to-array.js:3:19`, but there is no such file:

```js
var _isIterable = require("babel-runtime/core-js/is-iterator")["default"];
```

**Comment by deepsweet** [comment]
:+1:

**Comment by sebmck** [comment]
Fixed as of 5.1.2

**Comment by dozoisch** [comment]
:+1:  Oh that's why it wasn't working thanks for the quick fix.
